src/site/content/en/learn/css/flexbox/flex.assess.yml

```diff
@@ -1,4 +1,4 @@
-setLeader: Test your knowledge of flexbox
+setLeader: Test your knowledge of flexbox. (Note: The following questions all assume `flex-direction: row`)
```

Comment:
```suggestion
setLeader: Test your knowledge of flexbox: `display: flex`
```
what do you think of this? make it explicit that it's a default flex container in context?

Reply:
I like that.  It still opens the door to vagueness (but how many people other than me thought it was vague?).

To be the most specific, we could have a sub(-sub)-title that gives the full class definition

```
Test your knowledge of flexbox with the following example
```
```css
.container {
  display: flex;
}
```
<img width="696" alt="image" src="https://user-images.githubusercontent.com/5297725/122981965-528d8d00-d368-11eb-9b0d-81827c44a03f.png">


But now I feel like I'm stretching a bit.  This may be over-explaining it to the reader, and it may be confusing to specify things like it's in an *ltr* language.

Reply:
I think you're making a stronger case, and I agree that the current quiz is making too many assumptions about the reader 🙂 

the quiz's support contextual code snippets as a stimulus: thoughts on this?

![Kapture 2021-06-22 at 13 22 45](https://user-images.githubusercontent.com/1134620/122994253-1c96db80-d35d-11eb-8f03-6b1e79cb96a7.gif)

```yml
stimulus: |
  ```css
  .container {
    display: flex;
    direction: ltr;
  }
  ```
```

Reply:
💯👌

I'll add that

Reply:
@argyleink I didn't test it locally, but I followed another [example of `stimulus`](https://github.com/GoogleChrome/web.dev/blob/068563f3e4eee82a8e62fa25b2b9ea049acb92d0/src/site/content/en/accessible/labels-and-text-alternatives/self-assessment.assess.yml#L12-L18) on the site, so 🤞

Reply:
Netlify-bot says the preview deployment failed.  Did I mess something up?

Reply:
@argyleink lookin' good :)

Reply:
Last nitpick, we need to add the same `stimulus` to the third question, since the question specifically mentions "height"

Reply:
> Last nitpick, we need to add the same `stimulus` to the third question, since the question specifically mentions "height"

nice, i'm down. let's change it to: "If you want content size used for child items" and let the height answer continue to be wrong?

Reply:
A few of the `rationale`s still mention `height`, so I'll put the `stimulus` there too

Reply:
Updated